app/models/labware_creators/pooled_tubes_from_whole_tubes.rb

```diff
@@ -39,33 +38,28 @@ def barcodes=(input)
 
     # TODO: This should probably be asynchronous
     def available_tubes
-      @search_options = OngoingTube.new(purposes: [parent.purpose.uuid], include_used: false, states: ['passed'])
-      @search_results = tube_search.all(
-        Limber::Tube,
-        @search_options.search_parameters
-      )
+      @search_options = OngoingTube.new(purpose_names: [parent.purpose.name], include_used: false)
```

Comment:
Why remove the `states: ['passed']` part?

Reply:
The existing search functionality was actually ignoring the filter anyway, but it occurred to me that its probably quite useful to see tubes that are almost ready as well. A pending or started tube in this list is only one step away from being available.